### DISCUSSION TITLE
Old Beatmap Difficulty Sorter.

### DISCUSSION THREAD

**Discussion by HideakiAtsuyo** [ppy/osu#17892] [Ideas] [answered]
Before we was able to choose to sort "by Difficulty" and it was showing 1 star, 2 stars etc.. it should be an option in osu lazer too :)

**Comment by peppy** [comment, marked as answer]
it will be.

**Reply by frenzibyte** [reply]
plus you're referring to group modes, and is tracked in https://github.com/ppy/osu/issues/4919

**Reply by HideakiAtsuyo** [reply]
Nice to hear ! This game is very good ! :)
